config/src/configs/delegate.rs

```diff
@@ -107,6 +115,7 @@ impl DelegateConfigInner {
 }
 
 impl DelegateConfigInner {
+    pub const GAS: u64 = 300_000_000_000_000;
```

Comment:
In reality this amount is the maximum of gas units that can be set in a transaction.
```suggestion
    // 300 Tgas
    pub const GAS: u64 = 300_000_000_000_000;
```